### DISCUSSION TITLE
Position Unset

### DISCUSSION THREAD

**Discussion by uttampun44** [tailwindlabs/tailwindcss#12532] [Ideas]
I have not seen a position-unset element in the tailwind. I am looking for these new features because sometimes we have to unset the position. I hope you guys will work on this or tell me how I can contribute these features to Tailwind.

**Comment by elmirshidev** [comment]
it is very strange that this is not available in tailwind :D

**Comment by edriso** [comment]
I was looking for that, but remembered that `static` is the default position, so maybe that's better to not add a redundant class?
